turbopy/diagnostics.py

```diff
@@ -207,15 +208,23 @@ def __init__(self, owner: Simulation, input_data: dict):
         self.location = input_data["location"]
         self.field_name = input_data["field"]
         self.output = input_data["output_type"]  # "stdout"
+        self.csv = None
+        self.csv_input = self._input_data.get("filename", None)
```

Comment:
These lines are not needed. The csv output is taken care of by the `outputter`. See line 256.